config/routes.rb

```diff
@@ -537,6 +537,9 @@
     post "uploads" => "uploads#create"
     post "uploads/lookup-urls" => "uploads#lookup_urls"
 
+    post "uploads/generate-presigned-put" => "uploads#generate_presigned_put"
```

Comment:
I wonder if the `put` verb would be better here, considering it even has `presigned_put` in the URL!

Not super important. Just a thought!

Reply:
Yeah it's a weird one because this endpoint creates an `ExternalUploadStub` record at the same time and is not changing an existing resource like PUT is usually used for

Reply:
Sure, sounds good.